src/common/createCryptoWrapper.ts

```diff
@@ -207,25 +212,35 @@ export function createCryptoWrapper (lib: any): IVirgilCryptoWrapper {
 			encryptionKeys.forEach(({ identifier, key }: EncryptionKey) => {
 				cipher.addKeyRecipientSafe(identifier, key);
 			});
-			return cipher.encryptSafe(data, true);
+			const retVal = cipher.encryptSafe(data, true);
```

Comment:
Is there some possible errors occurs on `encryptSafe` or `addKeyRecipientSafe` methods like in decryptWithPassword? Maybe some malformed private key, or public key instead.

Reply:
not that I know of. If the key is created with `generateKeyPair` method, this should never fail. Anyway, my reasoning was that failing to encrypt a message is an exceptional situation caused by a developer's mistake, whereas failing to decrypt is not.